### PR TITLE
Improve TPS, part 2

### PR DESCRIPTION
**Pull request by roman-khimov**
### Problem

TPS

### Solution

Improve it.

### CODE REVIEW

File: pkg/consensus/payload.go [hunk 1]
```diff
@@ -224,15 +230,39 @@ func (p *Payload) DecodeBinaryUnsigned(r *io.BinReader) {
 	}
 }
 
+// GetSignedHash returns a hash of the payload used to verify it.
+func (p *Payload) GetSignedHash() util.Uint256 {
+	if p.signedHash.Equals(util.Uint256{}) {
+		p.createHash()
```
Comment (fyrchik) [inline pkg/consensus/payload.go:236]: Why do we ignore error here? (vs panic in `Hash()`)

File: pkg/core/storage/leveldb_store.go [hunk 2]
```diff
@@ -21,8 +22,9 @@ type LevelDBStore struct {
 // NewLevelDBStore returns a new LevelDBStore object that will
 // initialize the database found at the given path.
 func NewLevelDBStore(cfg LevelDBOptions) (*LevelDBStore, error) {
-	var opts *opt.Options // should be exposed via LevelDBOptions if anything needed
+	var opts = new(opt.Options) // should be exposed via LevelDBOptions if anything needed
 
+	opts.Filter = filter.NewBloomFilter(10)
```
Comment (fyrchik) [inline pkg/core/storage/leveldb_store.go:27]: How about an option in `*.yml` config?

File: pkg/rpc/server/server.go [hunk 3]
```diff
@@ -712,7 +712,7 @@ func (s *Server) getStorage(ps request.Params) (interface{}, *response.Error) {
 
 	item := s.chain.GetStorageItem(id, key)
 	if item == nil {
-		return nil, nil
```
Comment (fyrchik) [inline pkg/rpc/server/server.go:715]: Seems unrelated to this commit.

Comment (roman-khimov) [reply pkg/rpc/server/server.go:715]: In fact it is, previous scheme allowed the associated test to pass.

File: pkg/core/storage/leveldb_store.go (hunk 2 above)
Comment (roman-khimov) [reply pkg/core/storage/leveldb_store.go:27]: Meh. I had this though, bu it's so deep that I'm not sure it's really appropriate. But at the same time it affects memory consumption if nothing else.

File: pkg/consensus/payload.go (hunk 1 above)
Comment (roman-khimov) [reply pkg/consensus/payload.go]: Fixed.

**Review by AnnaShaleva — APPROVED** [review]
